src/main/java/mil/dds/anet/AnetApplication.java

```diff
@@ -277,6 +268,37 @@ public void run(AnetConfiguration configuration, Environment environment) {
             metricRegistry));
   }
 
+  private void runAccountDeactivationWorker(final AnetConfiguration configuration,
+      final ScheduledExecutorService scheduler, final AnetObjectEngine engine)
+      throws IllegalArgumentException {
+    // Check whether the application is configured to auto-check for account deactivation
+    if (configuration.getDictionaryEntry("automaticallyInactivateUsers") != null) {
+      // Check for any accounts which are scheduled to be deactivated as they reach the end-of-tour
+      // date.
+      final String intervalDictKey = "automaticallyInactivateUsers.checkIntervalInSecs";
+      final Integer accountDeactivationWarningInterval =
+          (Integer) configuration.getDictionaryEntry(intervalDictKey);
+
+      if (accountDeactivationWarningInterval == null) {
+        final String errorMsg =
+            "Missing or invalid value '" + intervalDictKey + "' in the configuration";
+        logger.error(errorMsg);
+        throw new IllegalArgumentException(errorMsg);
+      }
```

Comment:
If `checkIntervalInSecs` is _required_ in the schema (and with proper type), this check should be unnecessary.